tests/test_logging.py

```diff
@@ -0,0 +1,87 @@
+import re
+
+
+def _remove_timestamps(string):
+    return re.sub(
+        r"[A-Z][a-z][a-z] [A-Z][a-z][a-z] \d\d \d\d:\d\d:\d\d \d\d\d\d",
+        "<timestamp>",
+        string,
+    )
+
+
+def check_log(path, expected_content):
+    assert (
+        _remove_timestamps(path.read_text("utf-8")).expandtabs(12) == expected_content
```

Comment:
It would be possible to do `.expandtabs(8)` or just `.expandtabs()`, if `<timestamp>` was something shorter, e.g. `<time>`.